src/utils/auth/resetPassword.ts

```diff
@@ -1,6 +1,7 @@
-import makeRequest from '/src/utils/services/makeRequest';
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import makeRequest from 'src/utils/services/makeRequest';
 
-export const resetPassword = async (email, signURL, changeState) => {
+export const resetPassword = async (email: string, signURL: string, changeState: (arg0: string) => string) => {
```

Comment:
`changeState` and `arg0` could be more specific to make understanding what kind of function should be used here easier.